app/uk/gov/hmrc/vatapi/connectors/BaseConnector.scala

```diff
@@ -36,19 +36,19 @@ trait BaseConnector {
   private val logger = Logger("connectors")
 
   def httpGet[R <: Response](url: String, toResponse: HttpResponse => R)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[R] =
-    withAdditionalHeaders[R](url, withDesHeaders) {
+    withAdditionalHeaders[R](url, withDesHeaders()) {
       http.GET(url)(NoExceptionsReads, _, ec) map toResponse
     }
 
-  private def withDesHeaders: HeaderCarrier => HeaderCarrier = { hc =>
+  private def withDesHeaders(orgFlag: Boolean = false): HeaderCarrier => HeaderCarrier = { hc =>
     val newHc: HeaderCarrier = hc
       .copy(authorization = Some(Authorization(s"Bearer ${appContext.desToken}")))
       .withExtraHeaders(
         "Environment" -> appContext.desEnv,
-        "Accept" -> "application/json",
-        "Originator-Id" -> "DA_SDI"
+        "Accept" -> "application/json"
       )
 
+    if(orgFlag) newHc.withExtraHeaders("OriginatorID " -> "MDTP")
```

Comment:
Nice to have test in VatReturnsConnectorSpec for this (along the lines of ObligationsConnectorSpec that used an ArgumentCaptor for a get but for a post).